README.md

```diff
@@ -137,6 +137,7 @@ There are three types of tests for Lantern and three corresponding commands:
 |  `make lint` | Runs the R and golang linters |
 |  `make lint_go` | Runs the golang lintr |
 |  `make lint_R` | Runs the R lintr |
+| `make json_export` | Exports the history of the endpoint data as a JSON file |
```

Comment:
Should add example usage -- took me a minute to figure out that i needed to provide a `file=` parameter eg: make json_export file="./testjson.json"